lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -2165,6 +2155,81 @@ def jsonrpc_claim_list(self, name):
         claims = yield self.session.wallet.get_claims_for_name(name)
         defer.returnValue(claims)
 
+    @AuthJSONRPCServer.auth_required
+    @defer.inlineCallbacks
+    def jsonrpc_claims_in_channel_list(self, page=0, page_size=10, uri=None, uris=[]):
```

Comment:
we talked about this being in `claim_list`. we should not have a `claims_in_channels_list` RPC command. its dirty.